### PR TITLE
added linking to card on landing page

### PR DESCRIPTION
**Pull request by Manan-dev**
I had to link the "hybrid event details" text to the "park" card because if it added the id to hybrid event details card then it was scrolling too far.

### CODE REVIEW

**Comment by Manan-dev** [comment]
nevermind, I reordered the cards and  I added the id to "row" div so it scrolls correctly now.

**Comment by Manan-dev** [comment]
nevermind again, it breaks on mobile

**Comment by andreicozma1** [comment]
> nevermind again, it breaks on mobile

If needed we can just scroll to the top of the entire FAQ section.
Is the mobile issue still there?

**Comment by Manan-dev** [comment]
Ok I made it so it scrolls to the top of FAQ section rather than the card itself

File: index.html [hunk 1]
```diff
@@ -109,6 +109,7 @@ <h1 class="brand-heading"><span class="light orange">Vol</span><span class="eure
                     <h3>October 29th -  31st</h3>
 
                     <h3><a class="page-scroll" href="#venue">Min H. Kao Building<br>UTK</a></h3>
```
Comment (andreicozma1) [inline index.html:111]: Should hybrid event details be above the venue? Also should we clarify the venue being for in person somehow?

**Review by andreicozma1 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Manan-dev) [reply index.html:111]: Should i put "(for in person)" besides the title "venue"?

**Pull request merged**